chirp/drivers/alinco_dr735t.py

```diff
@@ -283,6 +283,10 @@ def find_key_in(d: dict, target_val):
             _mem.power_index = self.POWER_MAP.index(
                 mem.power) if mem.power in self.POWER_MAP else 0
             _mem.skip = 0x01 if mem.skip == "S" else 0x00
+            # set favorite, bell to 1 if it's ANYTHING not the defaults or 0
+            _mem.favorite = 0x01 if _mem.favorite not in [0x00, 0xFF] else 0x00
```

Comment:
Are you sure this works? `_mem.favorite` is a `dataElement` instance not an integer so I'm not sure this will ever be true.

Reply:
Also, would you prefer to add these to the extras so you can control them? Would also fix the defaulting... Not sure how important they are, but it's not hard to do and I can show you how if you want.

Reply:
I would be happy to add them to extras since I already plumbed for some of the extras in the original driver PR. I'll work on that. Do you prefer I close this PR and open a new one with this bugfix and the extras or just update this one?

Reply:
No, please just force push to this one, thanks.

LMK if you need help, but there should be some examples to copy in that driver already.